packages/atomic/src/components/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -166,19 +171,17 @@ export class AtomicSearchInterface {
 
   @Watch('language')
   public updateLanguage() {
-    if (!this.engineIsCreated(this.engine)) {
-      return;
+    if (this.engineIsCreated(this.engine)) {
```

Comment:
Why the change in this function ? I personally like the early return pattern, helps readability.